freqtrade/rpc/api_server/ws/channel.py

```diff
@@ -24,6 +24,8 @@ def __init__(
         self,
         websocket: WebSocketType,
         channel_id: Optional[str] = None,
+        drain_timeout: int = 3,
+        throttle: float = 0.01,
```

Comment:
I'm not sure i like these as arguments.
It makes you think "configurability per channel" - but it really isn't.

for a constant, i think setting them directly in __init__ would be better.

Reply:
This is really no different than the `serializer_cls` parameter as well. I don't know about being configurable right now, but potentially in the future it's a possibility. What do you think?